ansible/roles/ocp4-workload-rhtr-iot-managed/tasks/pre_workload.yml

```diff
@@ -0,0 +1,67 @@
+---
+# Implement your Pre Workload deployment tasks here
+
+- name: Ensuring AWS region is defined
+  fail:
+    msg: "This workload requires aws_region to be defined. Exiting..."
+  when: (aws_region is not defined) and (ocp_workload_test is undefined)
+
+- name: usercount debug
+  debug:
+    msg: "Debugging num_users {{ num_users }}"
+
+- name: create usernames
+  set_fact:
+    users: "{{ users | default([]) + ['user'+item | string] }}"
+  loop: "{{ range(1,((num_users | int) + 1)) | list }}"
+
+# Figure out paths
+- name: extract api_url
+  shell: oc whoami --show-server
```

Comment:
Please avoid using shell module when possible. You can get the API endpoint using the `k8s_info` module. What you need will be in `.status.apiServerURL`.

```
k8s_info:
  api_version: config.openshift.io/v1
  kind: Infrastructure
  name: cluster
```